runbotics/runbotics-desktop/src/action/file/file.action-handler.ts

```diff
@@ -61,11 +63,24 @@ export default class FileActionHandler extends StatelessActionHandler {
     }
 
     async createFile(input: FileCreateFileActionInput): Promise<FileCreateFileActionOutput> {
-        try {
-            fs.closeSync(fs.openSync(`${input.path}`, 'a'));
-        } catch (err) {
-            throw Error(err);
-        }
+        if(fs.existsSync(input.path)) {
+            switch (input.conflict) {
+                case 'Overwrite': 
+                    await createNewFile(input.path);
+                    return 'File overwritten successfully';
+                case 'Extend name': {
+                    const newFilePath = getUniqueFileName(input.path);
+                    await createNewFile(newFilePath);
+                    return 'File with extended name created successfully';
+                }
+                case 'Throw Error':
+                    throw Error('File with the same name already exists');
+                default: 
+                    await createNewFile(input.path);
```

Comment:
leaving the success as default case can lead to hard debugging - there won't be error message and the log will say everything went successful, but the thing we want won't happen